src/NewCommand.php

```diff
@@ -159,7 +159,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $this->validateDatabaseOption($input);
         $this->validateStackOption($input);
 
-        $name = $input->getArgument('name');
+        $name = mb_rtrim($input->getArgument('name'), '/\\');
```

Comment:
This change breaks the minimum required PHP version. This function is introduced on PHP 8.4